### PR TITLE
chore(eventing): pass args to container

### PR DESCRIPTION
**Pull request by sinhaashish**

### CODE REVIEW

File: chart/templates/mayastor/obs/obs-callhome-deployment.yaml [hunk 1]
```diff
@@ -53,6 +53,9 @@ spec:
         {{- if .Values.obs.callhome.enabled }}
         - name: obs-callhome-stats
           image: "{{ .Values.image.registry }}/{{ .Values.image.repo }}/{{ .Chart.Name }}-obs-callhome-stats:{{ default .Values.image.tag .Values.image.repoTags.extensions }}"
+          args:
+            - "-n {{- .Release.Namespace }}"
+            - "-r {{- .Release.Name }}"
```
Comment (tiagolobocastro) [inline chart/templates/mayastor/obs/obs-callhome-deployment.yaml:58]: missing the mbus_url as well?

Comment (sinhaashish) [reply chart/templates/mayastor/obs/obs-callhome-deployment.yaml]: added

File: chart/templates/mayastor/obs/obs-callhome-deployment.yaml [hunk 2]
```diff
@@ -53,6 +53,10 @@ spec:
         {{- if .Values.obs.callhome.enabled }}
         - name: obs-callhome-stats
           image: "{{ .Values.image.registry }}/{{ .Values.image.repo }}/{{ .Chart.Name }}-obs-callhome-stats:{{ default .Values.image.tag .Values.image.repoTags.extensions }}"
+          args:
+            - "--namespace={{ .Release.Namespace }}"
+            - "--release-name={{ .Release.Name }}"
+            - "--mbus-url=nats://mayastor-nats:4222"
```
Comment (tiagolobocastro) [inline chart/templates/mayastor/obs/obs-callhome-deployment.yaml:59]: hmm should be {{ .Release.Name }}-nats ?

File: call-home/src/bin/stats/main.rs [hunk 3]
```diff
@@ -29,13 +29,17 @@ mod store;
 #[clap(name = utils::package_description!(), version = utils::version_info_str!())]
 struct Cli {
     /// The url for mbus.
-    #[clap(long, default_value = "nats://mayastor-nats:4222")]
+    #[clap(long, default_value = DEFAULT_MBUS_URL)]
     mbus_url: String,
```
Comment (niladrih) [inline call-home/src/bin/stats/main.rs:33]: Should be a Url type?

**Review by niladrih — APPROVED** [review]
Left one nitpick comment.

File: call-home/src/bin/stats/main.rs (hunk 3 above)
Comment (sinhaashish) [reply call-home/src/bin/stats/main.rs:33]: Function message_bus_init accepts a string so  string is the input 
```
/// Initialise the Nats Message Bus.
pub async fn message_bus_init(server: &str) -> impl crate::Bus {
    NatsMessageBus::new(server).await
} 
```

File: chart/templates/mayastor/obs/obs-callhome-deployment.yaml (hunk 2 above)
Comment (sinhaashish) [reply chart/templates/mayastor/obs/obs-callhome-deployment.yaml]: done

**Review by tiagolobocastro — APPROVED** [review]

**Comment by sinhaashish** [comment]
bors merge

**Pull request merged**